configs/yliyun.json

```diff
@@ -0,0 +1,18 @@
+{
+  "index_name": "yliyun",
+  "start_urls": [
+    "http://167.ylicloud.com/docs/index.html",
+    "http://167.ylicloud.com/docs"
+  ],
+  "stop_urls": [],
+  "selectors": {
+    "lvl0": "#content h1",
+    "lvl1": "#content h2",
+    "lvl2": "#content h3",
+    "lvl3": "#content h4",
+    "lvl4": "#content h5",
+    "text": "#content p, #content li"
+  },
+  "user_agent": "Googlebot",
```

Comment:
What is this for?

Reply:
Sometime body is empty with our regular user_agent

Reply:
Is that happening on all configs or only this one? If it's happening on several configs it might be something we need to fix on a higher level in DocSearch and not in the config.

If the default Scrappy user-agent is rejected I would suggest we either pass "DocSearch" as a UA and ask them to authorize it (this will be more fair than masquerading as Googlebot).

Reply:
Just few ones so far. Good idea

Reply:
I added an issue in https://github.com/algolia/docsearch-scraper/issues/404 to track this